templates/Coconut.xctemplate/ViewModelNetworking/___FILEBASENAME___ViewModel.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 class ___VARIABLE_sceneName___ViewModel: BaseViewModel { 
     var navigationService: NavigationService!
     var networking: AlamofireNetworking!
+    var parser: Parser!
     
     required init() {}
```

Comment:
@fajdof zas imamo 3 doslovno ista fajla lol

Reply:
Because of the unique way Xcode Templates work